packages/toast/api.js

```diff
@@ -1,161 +1,58 @@
-import { toaster as c } from '@fabric-ds/component-classes';
```

Comment:
The api methods `toast`, `updateToast` and `removeToast` provide a light wrapper with defaults around the `FabricComponentContainer`